sklearn/isotonic.py

```diff
@@ -190,6 +190,12 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     X_max_ : float
         Maximum value of input array `X_` for right bound.
 
+    X_thresholds_ : array-like of shape (n_thresholds,)
+        De-duplicated ascending `X` values suitable to interpolate the y = f(X) monotonic function.
+
+    y_thresholds_ : array-like of shape (n_thresholds,)
+        De-duplicated ascending `y` values suitable to interpolate the y = f(X) monotonic function.
```

Comment:
The continuous integration bots complain about the PEP8 code style that limits the lines to 80 columns max.